Instructions/Labs/AZ-204_05_lab_ak.md

```diff
@@ -429,7 +429,7 @@ In this exercise, you created a .NET console application to display a machine’
 
 1.  From the **ContainerCompute** blade, select the container registry that you created earlier in this lab.
 
-1.  From the **Container Registry** blade, select **Update**.
+1.  From the **Overview** blade, select **Update**. (it has an icon next to it that looks like a gear).
```

Comment:
Unfortunately, we can't say "looks like" as our instructions are required to be inclusive of users with visual impairments. Their screenreader will only indicate that this is the "Update" option.